src/main/java/com/czertainly/core/messaging/listeners/NotificationListener.java

```diff
@@ -37,6 +83,12 @@ public void processMessage(NotificationMessage notificationMessage) {
                     case TEXT -> {
                         NotificationDataText data = (NotificationDataText) notificationMessage.getData();
                         sendInternalNotifications(data.getText(), data.getDetail(), notificationMessage);
+                        if (notificationTypeStringMap != null) {
```

Comment:
Those ifs are duplicates, it is same for every case.

Reply:
I think the best way to fix it is to make the getNotificationSettings method return empty hash set instead of null when there are no settings, can I change it in this pull request?